analyzer/server.go

```diff
@@ -47,9 +51,15 @@ type Server struct {
 	GraphServer         *graph.Server
 	FlowMappingPipeline *mappings.FlowMappingPipeline
 	Storage             storage.Storage
+	FlowTable           *flow.FlowTable
+}
+
+func (s *Server) flowExpire(f *flow.Flow) {
+	s.FlowTable.Remove(f)
```

Comment:
I think there is an issue here, I'll try to explain what I understood :)

1. flowExpire is a callback that is called by AsyncExpire
2. flowExpire is calling Remove which is going to delete the flow
3. then AsyncExpire is going to delete the flow as well ?